dom/events/Event-dispatch-click.html

```diff
@@ -116,4 +116,57 @@
   })
   input.dispatchEvent(clickEvent)
 }, "redispatch during post-click handling")
+
+async_test(function(t) {
+  var input = document.createElement("input")
+  input.type = "checkbox"
+  dump.appendChild(input)
+  var child = input.appendChild(document.createElement("input"))
```

Comment:
Just a note from IRC for posterity:
```
09:40 < foolip> annevk: but why is the disabled test using nested input elements, is there some significance to that?
09:40 < annevk> foolip: yeah, to make sure the disabled input is not skipped when finding an element with activation
09:41 < foolip> annevk: oh, so it could have been any parent with activation behavior
09:41 < annevk> Yeah
```